packages/rpc-core/src/__tests__/unix-timestamp-test.ts

```diff
@@ -0,0 +1,31 @@
+import { assertIsUnixTimestamp } from '../unix-timestamp';
+
+describe('number', () => {
```

Comment:
I don't think this describe block is necessary. You're just describing what the assert function does.